index.js

```diff
@@ -66,48 +66,42 @@ function loader (registryOrVersion) {
     }
 
     static toNotch (item, serverAuthoritative = true) {
-      if (registry.supportFeature('itemSerializationAllowsPresent')) {
-        if (item == null) return { present: false }
-        const networkItem = {
-          present: true,
-          itemId: item.type,
-          itemCount: item.count
-        }
-        if (item.nbt && Object.keys(item.nbt.value).length !== 0) {
-          networkItem.nbtData = item.nbt
-        }
-        return networkItem
-      } else if (registry.supportFeature('itemSerializationUsesBlockId')) {
-        if (item == null) return { blockId: -1 }
-        const networkItem = {
-          blockId: item.type,
-          itemCount: item.count,
-          itemDamage: item.metadata
-        }
-        if (item.nbt && Object.keys(item.nbt.value).length !== 0) {
-          networkItem.nbtData = item.nbt
+      const hasNBT = item.nbt && Object.keys(item.nbt.value).length > 0
+
+      if (registry.type === 'pc') {
+        if (registry.supportFeature('itemSerializationAllowsPresent')) {
+          if (item == null) return { present: false }
+          const networkItem = {
+            present: true,
+            itemId: item.type,
+            itemCount: item.count
+          }
+          if (hasNBT) networkItem.nbtData = item.nbt
```

Comment:
this should not be necessary, it will just be set as undefined if assigned to networkItem. You can just do 

```
nbtData: hasNBT ? item.nbt : undefined
```

inside the object.

Reply:
I set it because it still sets `nbtData` to undefined, and that caused the tests to fail because they're expecting the field not to be there. I'll change the tests then.